component/input/input.html

```diff
@@ -5,74 +5,83 @@
+                        <h3 class="input-subtitle">Text Input</h3>
+                    </div>
+                    <div class="number-input box">
+                        <input type="number" placeholder="Number Input Box" class="input-box">
+                        <h3 class="input-subtitle">Number Input</h3>
+                    </div>
+                    <div class="password-input box">
+                        <input type="password" placeholder="Password Input Box" class="input-box">
+                        <h3 class="input-subtitle">Password Input</h3>
+                    </div>
+                </div>
             </div>
-            <br>
-            <iframe
+            <iframe class="code"
                 src="https://carbon.now.sh/embed?bg=rgba%28255%2C59%2C48%2C1%29&t=blackboard&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=143%25&si=false&es=2x&wm=false&code=%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522text-input%2520box%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cinput%2520type%253D%2522text%2522%2520placeholder%253D%2522Text%2520Input%2520Box%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ch3%2520class%253D%2522input-subtitle%2522%253EText%2520Input%253C%252Fh3%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%2520%2520%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522number-input%2520box%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cinput%2520type%253D%2522number%2522%2520placeholder%253D%2522Number%2520Input%2520Box%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ch3%2520class%253D%2522input-subtitle%2522%253ENumber%2520Input%253C%252Fh3%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522password-input%2520box%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cinput%2520type%253D%2522password%2522%2520placeholder%253D%2522Password%2520Input%2520Box%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ch3%2520class%253D%2522input-subtitle%2522%253EPassword%2520Input%253C%252Fh3%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E"
-                style="width: 774px; height: 409px; border:0; transform: scale(1); overflow:hidden;"
                 sandbox="allow-scripts allow-same-origin">
             </iframe>
-            <div class="radio-checkbox-inputconatainer">
```

Comment:
Better to correct the typo for 'inputcontainer' instead of 'inputconatainer' as it may be used by others

Reply:
Hey i have already remove it
this red show removed code

Reply:
OK. Then it's Good to go.